graph.js

```diff
@@ -197,10 +197,14 @@ app
             logger.fatal('Error deploying the subgraph:', jsonRpcError.message)
           }
           if (!requestError && !jsonRpcError) {
-            logger.status(
-              'Deployed to Graph node:',
-              path.join(requestUrl.toString(), cmd.subgraphName)
-            )
+            logger.status('Deployed to Graph node:', requestUrl.toString())
```

Comment:
Let's also simplify this to `Deployed successfully`, so we don't repeat the URL too many times.